Major Update (#268)
NilPointer-Software/bdfd-wiki: src/bdscript/isMentionable.md

$isMentionable[roleID]
```

### Parameters

> - `roleID` `(Type : Snowflake || Flag : Required)` : The role which the bot checks is mentionable or not.

Reviewer: ```suggestion
- `roleID` `(Type: Snowflake || Flag: Required)`: The role to check for its mentionable status.
```
